product_supplierinfo_discount/README.rst

```diff
@@ -47,6 +47,7 @@ Contributors
 ------------
 
 * Pedro M. Baeza <pedro.baeza@serviciosbaeza.com>
```

Comment:
To access prices and discounts in the supplier info view, you have to enable the
 option "Manage pricelist per supplier" in *Configuration > Purchases*